common/context.go

```diff
@@ -27,6 +30,32 @@ func GetString(ctx context.Context, key ContextKey) (value string) {
 	return
 }
 
+// GetBasic gets basics from the context if any
+func GetBasic(ctx context.Context, key string) (value interface{}) {
+	if ctx != nil {
+		if basics, ok := ctx.Value(ctxBasics).(Basics); ok {
+			value, _ = basics[key]
+		}
+	}
+	return
+}
+
+// GetBasics gets basics from the context if any
+func GetBasics(ctx context.Context) (value Basics) {
+	if ctx != nil {
+		value, _ = ctx.Value(ctxBasics).(Basics)
+	}
+	return
+}
+
+// SetBasics returns a copy of parent context with extras added into it
```

Comment:
SetBasics returns a copy of parent context with ~extras~ **basics** added into it